lib/all-badge-examples.js

```diff
@@ -1016,6 +1016,22 @@ const allBadgeExamples = [
         previewUri: '/gratipay/project/shields.svg'
       },
       {
+        title: 'Liberapay receiving',
+        previewUri: '/liberapay/Liberapay/receives.svg'
```

Comment:
Could you flip these around for consistency with other badges? In other words, put the noun like `receives` before the project name, instead of after.

Also, it's usually clearer to give an example that does not refer back to the service. If you could pick another org than Liberapay itself, that would be helpful.